js/data/odata/context.js

```diff
@@ -19,7 +16,7 @@ require('./query_adapter');
 */
 const ODataContext = Class.inherit({
```

Comment:
`export default const ODataContext = Class.inherit({`

Reply:
We cannot use `export default const` (this construction does not applicable)
We should use `export default Class.inherit({...})` or define `const ODataContext = ...` and make `export default ODataContext` (as it is now)